content/en/blog/_posts/2024-03-18-windows-ops-readiness.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+title: "Introducing the Windows Operational Readiness Specification"
+date: 2024-03-18
+slug: intro-windows-ops-readiness
+---
+
+**Authors:** Jay Vyas (Tesla), Amim Knabben (Broadcom), and Tatenda Zifudzi (AWS)
+
+# Introducing the Windows Operational Readiness Specification
+
+Since Windows support [graduated to stable](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/release-notes-draft.md) with 1.14 in 2019, the capability to run Windows workloads has consistently been a major differentiator for Kubernetes distributions in the enterprise. However, with more Windows workloads being migrated to Kubernetes and new Windows features being continuously released, it became challenging to test Windows worker nodes in an effective and standardized way.
+
+Some notable examples brought to the attention of SIG Windows were:
+
+- An issue with [loadBalancerSourceRanges](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#:~:text=loadBalancerSourceRanges) functionality not operating correctly on Windows nodes, detailed in a GitHub issue [kubernetes/kubernetes#120033](https://github.com/kubernetes/kubernetes/issues/120033).
```

Comment:
Even though Chrome supports that kind of fragment identifier, I wouldn't include that link (it's not a standardized mechanism, and the single-page API reference you've linked to is one we plan to supersede).
```suggestion
- An issue with load balancer source address ranges functionality not operating correctly on
  Windows nodes, detailed in a GitHub issue:
  [kubernetes/kubernetes#120033](https://github.com/kubernetes/kubernetes/issues/120033).
```